pkg/controller/migmigration/task.go

```diff
@@ -649,16 +649,18 @@ func (t *Task) Run() error {
 				return liberr.Wrap(err)
 			}
 		}
+		t.Owner.Status.FindStep(t.Step).Message = dvm.Status.PhaseDescription
```

Comment:
Instead of managing the step information outside of the default pipeline helper functions, how about adding a new function `t.setStepDescription(desc string)`. This will remove the need of marking step completed and all other custom logic written for this particular step.